.env.example

```
@@ -24,3 +24,7 @@ CLOUDINARY_API_SECRET="Add your secret"
 #If working with 1v1 quiz
 REDIS_URL=
 REDIS_TOKEN=
+
+# Clist Username and Clist_API_key for past contests
+CLIST_USERNAME="Add your Clist Api username"
+CLIST_API_KEY="Add your key"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)